app/screens/send-bitcoin-screen/payment-details/onchain.ts

```diff
@@ -275,12 +275,12 @@ export const createAmountOnchainPaymentDetails = <T extends WalletCurrency>(
     canGetFee: false,
   }
 
-  let sendPayment: SendPayment
+  let sendPaymentSelector: SendPaymentSelector
   let getFee: GetFee<T>
 
   if (sendingWalletDescriptor.currency === WalletCurrency.Btc) {
-    sendPayment = async (sendPaymentFns) => {
```

Comment:
i have also renamed `sendPaymentFns` to `paymentMutations` just to clarify the later are just the mutation we are injecting.